intel-extension-for-pytorch/Distributed/KubeRay_Getting_Started.ipynb

```diff
@@ -0,0 +1,331 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a20d3ca9-2f58-43b0-b7aa-2e80eb588ad0",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "This notebook demonstrates how you can utilize the Intel Extension for PyTorch to optimize distributed workloads  on Intel hardware with RedHat OpenShift AI and KubeRay operator. For this demo we finetune a Large Language Model from HuggingFace tranformers on 2 or more nodes. The notebook uses codeflare SDK to create a Ray cluster and launch a distributed training job on it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cbf33887-b3ad-4c0c-b5d7-7a3fd4ffa1fa",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Install CodeFlare SDK"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "60ae1703-9ebe-4fe5-8278-bede84609a8e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! pip install codeflare-sdk==0.14.1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27db73de-d7ee-4587-91b7-25eaf4df51e5",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Importing necessary codeflare SDK modules"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bded3130-c62c-4773-94bd-b008e2564900",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from codeflare_sdk.cluster.cluster import Cluster, ClusterConfiguration\n",
+    "from codeflare_sdk.cluster.auth import TokenAuthentication\n",
+    "from codeflare_sdk.job.ray_jobs import RayJobClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "262b0fc6-ca66-4c82-a439-65d03dc699b5",
+   "metadata": {},
+   "source": [
+    "## Authenticating login to the OCP cluster\n",
+    "\n",
+    "\n",
+    "**NOTE: Please fill in the value of variable auth_token below.**\n",
+    "\n",
+    "To find out the token please use the RedHat OpenShift Container Platform's online console."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc5c7791-a2fc-4b2d-af40-6b65ec8b976e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "auth_token = \"XXXX\"\n",
+    "api_server = \"XXXX\"\n",
+    "auth = TokenAuthentication(\n",
+    "    token=auth_token,\n",
+    "    server=api_server,\n",
+    "    skip_tls=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a50572b-f237-4ca4-a2d0-91623504f846",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "auth.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44dbaf70-3622-464f-8f0c-2e8df7a91611",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Launch a Ray cluster using Codeflare SDK."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e7ba13aa-5608-489d-bbf3-e2d55f340295",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "registry = \"XXXX\"\n",
```

Comment:
I'd add a note here about user's responsibility to provide registry.
Also should we move all these into just one cell and ask user to provide them.